src/main/java/org/cloudbees/literate/jenkins/LiterateEnvironmentBuild.java

```diff
@@ -202,7 +202,7 @@ protected Result doRun(final BuildListener listener) throws Exception, RunnerAbo
             ProjectModel model = projectModelAction.getModel();
             Map<Class<? extends Publisher>, List<Agent<? extends Publisher>>> agents = new LinkedHashMap<Class<?
                     extends Publisher>, List<Agent<? extends Publisher>>>();
-            for (Agent agent : Jenkins.getInstance().getExtensionList(Agent.class)) {
+            for (Agent agent : ExtensionList.lookup(Agent.class)) {
```

Comment:
That is the difference for now. (In the future, `ExtensionList.lookup` could be improved to return a meaningful result even from unit tests, as [here](https://github.com/jenkinsci/workflow-plugin/blob/5189e3205f63e7d094180109db3ee6f7d3d37d7d/step-api/src/main/java/org/jenkinsci/plugins/workflow/structs/DescribableHelper.java#L435-447).)